python/ct/crypto/in_memory_merkle_tree.py

```diff
@@ -0,0 +1,125 @@
+"""In-memory Merkle Tree
```

Comment:
C6108:  1: One-line docstring summary should end with ".", "?", or "!" [g-short-docstring-punctuation]